crates/sui-indexer-alt/src/pipeline/processor.rs

```diff
@@ -26,7 +27,7 @@ pub trait Processor {
     const FANOUT: usize = 10;
 
     /// The type of value being inserted by the handler.
-    type Value: Send + Sync + 'static;
+    type Value: Send + Sync + 'static + FieldCount;
```

Comment:
I think it's more correct to add a construct on the `concurrent::Handler` trait than here, like:

```rust
trait Handler: Processor<Value: FieldCount> {
```